lib/livebook/storage/ets.ex

```diff
@@ -0,0 +1,110 @@
+defmodule Livebook.Storage.Ets do
+  @moduledoc """
+  Ets implementation of `Livebook.Storage` behaviour.
+
+  The module is supposed to be started just once as it
+  is responsible for managing a single global named ets table.
+
+  `insert` and `delete` operations are supposed to be called using the genserver
+  while all the lookups can be performed by directly accessing the named table.
+  """
+  @behaviour Livebook.Storage
+
+  @table_name :__livebook_storage__
+
+  use GenServer
+
+  @impl Livebook.Storage
+  def fetch(namespace, entity_id) do
+    @table_name
+    |> :ets.lookup({namespace, entity_id})
+    |> case do
+      [] ->
+        :error
+
+      entries ->
+        entries
+        |> Enum.map(fn {_key, attr, val, _timestamp} ->
+          {attr, val}
```

Comment:
I just realized perhaps we could store `{attr, val}` as a tuple. This way this could be a `:ets.lookup_element/2`. WDYT?

Reply:
I think that we can't really use `lookup_element` due to this quirk from documentation `If no object with key Key exists, the function exits with reason badarg.` which is pretty bad.

Reply:
Yeah, we typically do this:

```elixir
try do
  :ets.lookup_element(@table, key, 2)
catch
  _ -> []
end
```

Reply:
To be honest I think that keeping attribute key and value separately looks more clear and enables to operate on them more freely (an example would be the `select_delete` that I've added).